dbt/parsely_dpl_redshift/redshift.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import
+
+import datetime as dt
+import argparse
+import subprocess
+import os
+
+import parsely_raw_data.redshift as prd_redshift
+import parsely_raw_data.utils as utils
```

Comment:
You can just write `from parsely_raw_data import utils` and it will have the same effect.